src/awala.ts

```diff
@@ -33,3 +48,117 @@ export async function createMemberBundleRequest(
     didSucceed: true,
   };
 }
+// string will be substituted with the actual error
+export async function generateMemberBundle(
```

Comment:
I don't think this function belongs in this file: It does, indeed, use the Awala middleware, but just like a [_backing service_](https://12factor.net/backing-services) -- much like MongoDB and the Knative broker (emitter).

The functionality it implements is agnostic of the `POST /awala` endpoint. It's more closely related to the _org member_, so I think we should create a new file like `src/memberBundle.ts` -- analogous to all the other business logic files in `src/*.ts`.

Reply:
This was moved to memberBundle.ts - https://github.com/relaycorp/veraid-authority/pull/119/files#diff-5006ad038ccf0df13d5fba8db8001b0fad11d2deb7c5c40fb830dc68e101ab0bR78